frontend/src/httpClient.tsx

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
```

Comment:
I have a pretty significant dislike for axios since it often has security vulnerabilities that need to be patched and it's just a wrapper on the browser's native fetch API (which is well supported on modern browsers). I would just use fetch.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch